pootle/templates/browser/index.html

```diff
@@ -74,7 +74,7 @@ <h3 class="top">{% trans "Translation Statistics" %}</h3>
       </div>
       {% if top_scorers %}
         <div class="summary-2-col">
-          <h3 class="top">{% trans "Top Contributors for the Last 30 Days" %}</h3>
+          <h3 class="top">{% trans "Top Contributors" %}, {% trans "Last 30 Days" %}</h3>
```

Comment:
There are quite a few localisation issues with this.  The usage of comma is locale specific.  The order is also locale specific.  Just translate it as one even with the comma.

Ironically I still think putting information like 30 days in a header is a kludge.